osmaxx-py/osmaxx/excerptexport/static/excerptexport/libraries/Leaflet.Spin/leaflet.spin.js

```diff
@@ -0,0 +1,41 @@
+L.SpinMapMixin = {
+    spin: function (state, options) {
+        if (!!state) {
+            // start spinning !
+            if (!this._spinner) {
+                this._spinner = new Spinner(options).spin(this._container);
+                this._spinning = 0;
+            }
+            this._spinning++;
```

Comment:
Can there be multiple reasons to spin at a given moment and you have to keep spinning until all of them are gone?